feedhenry/fh-js-sdk
# task(Upgrade Dependencies without major/break changes):  solve security vulnerabilities

## JIRA:
Link : https://issues.jboss.org/browse/FH-4373

## WHAT:
Upgrade the dependencies which don't have break changes. 

## HOW:
- Update fh-sync-js from 1.1.1 to 1.3.0
- Update loglevel from ~0.6.0 to ~1.6.1
- Update process from ~0.6.0 to ~0.11.10 
- Update underscore from ~0.6.0 to ~1.9.0 *
- Upgrade dev dependencies which don't have break changes which impact the test. 

PS.:
* The major version `1.0.0` of underscore don't have break changes, it is the version which starts to be official stable release!

## WHY
To Solve one or more security vulnerabilities by upgrade the dependencies which don't have break changes and technical debts. 

## Steps to test:
Check if the CI/Process finalized with success since it will install this lib and run all tasks and test with the all supported NodeJS versions.
